docs/appendices/release-notes/unreleased.rst

```diff
@@ -46,4 +46,10 @@ None
 Fixes
 =====
 
-- Fixed case sensitivity of unquoted column names inside ``ON CONFLICT`` clauses.
\ No newline at end of file
+- Fixed a regression introduced in 4.0.8 which could cause queries with an
+  explicit limit larger than 1000 to fail with a ``numHits must be > 0`` error.
+
+- Fixed an resiliency issue on snapshot creation while dynamic columns are
+  created concurrently which may result in incompatibility problems on restore.
+
+- Fixed case sensitivity of unquoted column names inside ``ON CONFLICT`` clauses.
```

Comment:
Not sure why the diff shows the snapshot creation thing as added, it is already on 4.0 - see https://github.com/crate/crate/blob/4.0/docs/appendices/release-notes/unreleased.rst#id2